boards/st/nucleo_h745zi_q/nucleo_h745zi_q_stm32h745xx_m7.yaml

```diff
@@ -16,6 +16,7 @@ supported:
   - counter
   - i2c
   - pwm
+  - can
```

Comment:
By adding can in the list, it will be tested automatically, which will fail as node is disabled by default.

Reply:
I will change to pins set mentioned by @FRASTM